nikola/plugins/basic_import.py

```diff
@@ -166,7 +166,7 @@ def write_metadata(filename, title, slug, post_date, description, tags, **kwargs
         with io.open(filename, "w+", encoding="utf8") as fd:
             data = {'title': title, 'slug': slug, 'date': post_date, 'tags': ','.join(tags), 'description': description}
             data.update(kwargs)
-            fd.write(utils.write_metadata(data))
+            fd.write(utils.write_metadata(data, metadata_format='nikola', comment_wrap=False))
```

Comment:
Ah, right. I misremembered `needs_config` — turns out a site always exists, but isn’t always required to be configured. I’ll fix that.